packages/edgeidentity/README.md

```diff
@@ -94,18 +97,201 @@ public class MainApplication extends Application implements ReactApplication {
 
+Refer to [reset identities api](https://github.com/emdobrin/aepsdk-react-native/tree/consent/packages/core#reset-identities)
+
+## Public Classes
+### IdentityMap
+
+Defines a map containing a set of end user identities, keyed on either namespace integration code or the namespace ID of the identity. The values of the map are an array, meaning that more than one identity of each namespace may be carried.
+The format of the IdentityMap class is defined by the [XDM Identity Map Schema](https://github.com/adobe/xdm/blob/master/docs/reference/mixins/shared/identitymap.schema.md).
+
+For more information, please read an overview of the [AEP Identity Service](https://experienceleague.adobe.com/docs/experience-platform/identity/home.html).
+
+```
+"identityMap" : {
+    "Email" : [
+      {
+        "id" : "user@example.com",
+        "authenticatedState" : "authenticated",
+        "primary" : false
+      }
+    ],
+    "Phone" : [
+      {
+        "id" : "1234567890",
+        "authenticatedState" : "ambiguous",
+        "primary" : false
+      },
+      {
+        "id" : "5557891234",
+        "authenticatedState" : "ambiguous",
+        "primary" : false
+      }
+    ],
+    "ECID" : [
+      {
+        "id" : "44809014977647551167356491107014304096",
+        "authenticatedState" : "ambiguous",
+        "primary" : true
+      }
+    ]
+  }
+```
+**Example**
+```javascript
+var map = new AEPIdentityMap();
+
+// Add an item
+var identityItem  = new AEPIdentityItem("user@example.com");
+map.addItem(identityItem, "Email");
+
+// Remove an item
+var identityItem  = new AEPIdentityItem("user@example.com");
+map.removeItem(identityItem, "Email");
+
+//Get a list of items for a given namespace
+var mamespacecheck = map.getIdentityItemsForNamespace("Email");
+
+//Get a list of all namespaces used in current IdentityMap
+var namespaces = map.getNamespaces();
+
+//Check if IdentityMap has no identities
+var hasNoIdentities = new Boolean(map.isEmpty());
+```
+
+### IdentityItem
+Defines an identity to be included in an IdentityMap.
+
+The format of the IdentityItem class is defined by the [XDM Identity Item Schema](https://experienceleague.adobe.com/docs/experience-platform/identity/home.html).
+
+**Example**
+```javascript
+
+// Initialize
+var item  = new AEPIdentityItem("identifier");
+
+var item  = new AEPIdentityItem("identifier", AEPAuthenticatedState.AUTHENTICATED, false);
+
+//Getters
+var id = item.id;
+var state = item.authenticatedState;
+var primary = item.primary
+```
+
+### AuthenticatedState
+Defines the state an Identity Item is authenticated for.
+
+The possible authenticated states are:
+* Ambiguous - the state is ambiguous or not defined
+* Authenticated - the user is identified by a login or similar action
+* LoggedOut - the user was identified by a login action at a previous time, but is not logged in now
+
+//TO DO: Add Syntax
+
+
+## Frequently Asked Questions (FAQ)
```

Comment:
I think we should remove this section and only point to the GitBook FAQ page, what do you think?

Reply:
I left the FAQ section with  referring to the gitbook link only.  Sounds good?